### PR TITLE
[Staging] Fix Flourish Viz download

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

Flourish viz was throwing errors on staging because media files are stored on s3

## Screenshots

Screenshot 2019-05-27 at 14 57 51

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by phillipahereza** [comment]
@karimkawambwa this kinda works now but there's a new error about the charset that's giving me a headache. Could you check it out

File: takwimu/views.py [hunk 1]
```diff
@@ -335,16 +337,24 @@ def get_context_data(self, *args, **kwargs):
 
 
 class FlourishView(APIView):
-    renderer_classes = (JPEGRenderer, StaticHTMLRenderer, CSSRenderer, JavaScriptRenderer,)
+    renderer_classes = (JPEGRenderer, FlourishHTMLRenderer, CSSRenderer, JavaScriptRenderer,)
 
     def get(self, request, *args, **kwargs):
         Document = get_document_model()
-        doc = get_object_or_404(Document, id=kwargs["document_id"])
+        # doc = get_object_or_404(Document, id=kwargs["document_id"])
```
Comment (karimkawambwa) [inline takwimu/views.py]: ☝️

File: takwimu/renderers.py [hunk 2]
```diff
@@ -23,8 +23,17 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
...
     media_type = 'image/jpeg'
     format = 'jpg'
-    charset = None
     render_style = 'binary'
 
     def render(self, data, media_type=None, renderer_context=None):
         return data
+
+
+class FlourishHTMLRenderer(StaticHTMLRenderer):
```
Comment (kilemensi) [inline takwimu/renderers.py]: What are we adding to `StaticHTMLRenderer`?

File: takwimu/renderers.py [hunk 3]
```diff
@@ -23,8 +23,17 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
 class JPEGRenderer(BaseRenderer):
     media_type = 'image/jpeg'
     format = 'jpg'
-    charset = None
```
Comment (kilemensi) [inline takwimu/renderers.py]: Does this cause any issues @phillipahereza?

File: takwimu/renderers.py (hunk 2 above)
Comment (phillipahereza) [reply takwimu/renderers.py]: @kilemensi we're removing the extra checks it was doing before rendering, they were throwing unicode errors

Comment (kilemensi) [reply takwimu/renderers.py]: Cool @phillipahereza... Just add a comment at the top of the class there why we're adding it for future reference.

Comment (phillipahereza) [reply takwimu/renderers.py]: @kilemensi done

**Pull request merged**